source/_integrations/comed_hourly_pricing.markdown

```diff
@@ -25,7 +25,11 @@ sensor:
   - platform: comed_hourly_pricing
     monitored_feeds:
       - type: five_minute
+        # Adds an offset of 10
+        offset: 10.0
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.